docs/app/pages/components/components-sections/timeline/timeline/timeline.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import 'chance';
+
+@Component({
+    selector: 'uxd-components-timeline',
+    templateUrl: './timeline.component.html'
+})
+@DocumentationSectionComponent('ComponentsTimelineComponent')
+export class ComponentsTimelineComponent extends BaseDocumentationSection {
+
+    constructor() {
+        super(require.context('./snippets/', false, /\.(html|css|js|ts)$/));
+    }
+
+    _developer = chance.name();
+
+    tickets = [{
+        color: 'critical',
+        shortDate: 'Mon 29th',
+        title: 'Addition of Timeline component',
+        date: new Date(2018, 0, 29, 11, 10),
```

Comment:
Could these dates be converted to `toLocaleString()` here? As currently we are calling this function in our view, which is best avoided where possible as the function will get called every time change detection runs making it a lot less efficient

Reply:
It was agreed that the 'date' property would be exposed as a Date.